data/test/scenarios/test_assert.cfg

```diff
@@ -79,7 +65,7 @@
 )}
 
 #####
-# API(s) being tested: unit_test.fail(),[fire_event]
+# API(s) being tested: [fire_event]
```

Comment:
The old version of this was right. It is testing `unit_test.fail()`, and checking that mechanism persists the failure when returning from the fired event.

Reply:
My thought here was that `unit_test.fail()` is already being tested in test_return.cfg, so removing it here narrows the number of APIs being tested, but that's true.